qmt_rgb2pct.1
.TH "QMT_RGB2PCT" "1" "Jan 2018" "" ""
.SH "NAME"
qmt_rgb2pct - a rgb2pct replacement that does not need python
.br

.SH "SYNOPSIS"
qmt_rgb2pct\fB \fR\fB [ -v \fR| \fB--version \fR] [ \fB-h \fR| \fB--help \fR] [ \fB-n \fR| \fB--ncolors \fR<number>] [ \fB-p \fR| \fB--pct \fR<filename>] [ \fB-s \fR| \fB--sct \fR<filename>] source traget
.br

.SH "DESCRIPTION"
Convert GeoTiff files with RGB or RGBA color scheme into a dithered GeoTiff with color table
.br

Main features:
.br
	- Optional read external color table
.br
	- Optional save external color table	
.br
	- Convert aplha band into a no data value	
.br

.SH "OPTIONS"
\fB-h, --help\fR
.br
	Displays this help.
.br

\fB-v, --version           \fR
.br
	Displays version information.
.br
	
.br
\fB-n, --ncolors <number>  \fR
.br
	Number of colors. (default: 255)
.br
	
.br
\fB-p, --pct <filename>    \fR
.br
	Input palette file for color table (*.vrt)
.br
	
.br
\fB-s, --sct <filename>    \fR
.br
	Save color table to palette file (*.vrt)
.br

\fBsource                  \fR
.br
	Source file.
.br
	
.br
\fBtarget                  \fR
.br
	Target file.
.br
.SH "SEE ALSO"
https://github.com/Maproom/qmapshack/wiki/DocMain
.br

.SH "AUTHOR"
qmt_rgb2pct was written and is currently maintained by Oliver Eichler \fB<oliver.eichler@gmx.de>\fR.
.br


changelog.txt
V1.XX.X
[QMS-68] Route Optimization
[QMS-132] DEMs: Make the scale range control cut off not only rendering but also loading or requesting the data.
[QMS-134] Rework track range selection from scratch
[QMS-135] Improved French translation
[QMS-140] Enhancements for OS X release
[QMS-146] Fix issues with built-in online maps
[QMS-148] Misplaced track info window
[QMS-151] Missing tooltip in route toolbar
[QMS-153] Improved French translation
[QMS-156] Request to unmount systemdrive on startup
[QMS-164] Workspace filter is not applied to newly loaded projects
[QMS-165] OSX: GIS files not loaded when clicked
[QMS-173] Misleading help text for DEM range slider
[QMS-178] Automate list of code contributors in about dialog
[QMS-183] Backward range selection causing small issues
[QMS-192] Fix URLs still pointing to the bitbucket wiki

V1.14.1
[QMS-37] Inconsistent use of time zones
[QMS-43] Individual filter for each project
[QMS-49] Add "elevation limit" to DEM to highlight elevation above the limit
[QMS-58] Windows Build and Installer: use PROJ 6.2
[QMS-59] Version information in window title does not contain VERSION_SUFFIX "development"
[QMS-64] Cleanup code in IUnit and subclasses
[QMS-73] Info of a geocache is not correctly shown in Copy Element Window
[QMS-79] Possibility to 'star' Waypoints
[QMS-80] Use ASAN to enhance code quality
[QMS-83] Application Crash
[QMS-89] Geo search hidden after restarting QMapShack
[QMS-100] Avoid whitespaces in project name and keywords
[QMS-108] Integrate Wiki Page as help dialog
[QMS-109] Direct link to geocaching logging page
[QMS-111] Geocaching logging page - only 1 server used
[QMS-116] Filter edit field lost
[QMS-118] Too much whitespace in extended search help window
[QMS-119] Better handling of disabled and archived geocaches
[QMS-125] Missing icon for not available geocache
[QMS-126] Wrong results searching for geocache status
[QMS-122] Windows: support for offline help

V1.14.0
[QMS-5] App crashes on using the "Change Start Point" filter
[QMS-6] Enhance overview detection for VRT maps
[QMS-8] Incorrect elevation for fit files from GPSMAP 66s
[QMS-12] Unify versions of all QMapShack tools
[QMS-13] Add support for Garmin Edge 500
[QMS-14] Drawing track: Alternate mode O A V T with a simple key with no modifiers
[QMS-18] Improved explanation of 'Date equals'
[QMS-19] Invalid GPX due to `::` in `ql` namespace
[QMS-20] Windows Start Menu - change links from bitbucket to github
[QMS-22] Aviation units: nm for distances and feet for altitude
[QMS-27] Error in workspace search for attributes
[QMS-31] Fix all links to Bitbucket in the code
[QMS-53] Unload Garmin `Archive` when folder is deflated
[QMS-54] Invalid point cannot be deactivated when tracks are read from the navigation device


------------------------------------------------------------------------
---------Restart issue numbering because of migration to GitHub---------
------------------------------------------------------------------------
V 1.13.2
[Issue #446] Advanced Filtering System
[Issue #491] Skip saving of geo search
[Issue #493] "Clone Waypoint and Move Clone" does not Respect Chosen Units
[Issue #488] Printing Preview
[Issue #494] BRouter segments download error
[Issue #499] Screen overflow German localisation on a Notebook with 1.600 x 900 pixel
[Issue #500]  Tab order in Filter Cycle dialog is confused
[Issue #498 ] BRouter setup issue (Windows)
Add: Computation of "Energy Use Cycling" (consumption) for cycling tours

V 1.13.1
[Issue #451] Make red border around a highlighted track an option
[Issue #452] Project Details: Add elevation info to entries with track point information.
[Issue #455] Profile Plot: Add option to show waypoint labels along line
[Issue #456] Clean up code regarding track info, attached wpt and hidden flag.
[Issue #460] Missing "Change Map View Name" action in toolbar
[Issue #465] Garmin IMG: Extend Tooltip
[Issue #467] Add activity color to track info and graph
[Issue #469] Reduce width of track detail dialog
[Issue #470] Crash when saving geocache
[Issue #471] Fix background color of textboxes and bubble for dark color themes
[Issue #477] BRouter setup issue (Windows)
[Issue #479] Unnecessary extra tag(s) when saving GPX file
[Issue #480] Graph not appearing when activating it
[Issue #481] Project Details: Add a roadbook with progress graphs
[Issue #466] Geocache detecting duplicates
[Issue #266] Scale on printed maps
[Issue #484] MainWindow slope display does not honor slope unit setting
[Issue #394] Added slope row in ruler table
Trk: Increase desity of arrows for tracks with a high track point frequency
BSD Fix: do not access QStringList.first() if list is empty

V 1.13.0
[Issue #298] Live GPS data
[Issue #430] Add waypoint summary to project details
[Issue #436] Route history is buggy and causes data loss when auto-routing is enabled
[Issue #433] Suppress ascent/descent if wrong
[Issue #438] Partially: Support for PROJ 6.0.0
[Issue #429] Add Geocaching Attributes
[Issue #434] Mega-fancy tour description on screen
[Issue #448] Project Details: Add track points with additional information to lists
[Issue #443] Garmin Map: Fixing elevation marks
[Issue #450] Waypoint: copy coordinate to clipboard

V 1.12.3
[Issue #428] natural=water not shown

V 1.12.2
[Issue #419] Add option to project to receive all newly created items
[Issue #417] Only one line of Geocache description
[Issue #421] Improve "edit" view of Geocaches
[Issue #425] Add option to disable device support
Fix: Garmin IMG: Fix type parser to accept sections with zero size
Fix: Plot: make left click more robust to small mouse moves

V 1.12.1
[Issue #390] Ruler: Add steps for distance and angle
[Issue #393] Some Linux dsitributions do not have planetsplitter but routino-planetsplitter
[Issue #395] Filter "Change Speed" Lacks Start Button
[Issue #399] Add filter line edit to map and DEM list
[Issue #400] Selecting item in workspace does not close screen options.

src/qmt_rgb2pct/locale/qmt_rgb2pct.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1">
<context>
    <name>CApp</name>
    <message>
        <location filename="../CApp.cpp" line="63"/>
        <source>Failed to open source file.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../CApp.cpp" line="68"/>
        <source>Raster band count of source file must be either 3 or 4.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../CApp.cpp" line="101"/>
        <source>Calculate optimal color table from source file</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../CApp.cpp" line="115"/>
        <source>Failed to create color table.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../CApp.cpp" line="123"/>
        <source>Failed to open file with palette.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../CApp.cpp" line="131"/>
        <source>Palette file does not have a single band with a color table</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../CApp.cpp" line="140"/>
        <source>The color table must not contain a &quot;no data&quot; value and it&apos;s size must not exceed 255 colors.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../CApp.cpp" line="167"/>
        <source>Save color table to: %1</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../CApp.cpp" line="203"/>
        <source>Failed to create target file.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../CApp.cpp" line="214"/>
        <source>Dither source file to target file</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../CApp.cpp" line="225"/>
        <source>Failed to dither file.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../CApp.cpp" line="240"/>
        <source>Apply alpha channel as no data value to target file</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../CApp.cpp" line="247"/>
        <source>Failed to read from alpha channel.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../CApp.cpp" line="253"/>
        <source>Failed to read from target file.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../CApp.cpp" line="267"/>
        <source>Failed to write to target file.</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>main</name>
    <message>
        <location filename="../main.cpp" line="96"/>
        <source>
Convert a map file with RGBA color coding to a color palette coding.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="99"/>
        <source>Source file.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="100"/>
        <source>Target file.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="104"/>
        <source>Number of colors. (default: 255)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="107"/>
        <source>Input palette file for color table (*.vrt)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="110"/>
        <source>Save color table to palette file (*.vrt)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="120"/>
        <source>There must be a source and destination file.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="143"/>
        <source>--ncolors must be an integer value less than 256</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
